src/goto-analyzer/static_verifier.cpp

```diff
@@ -307,7 +307,12 @@ bool static_verifier(
       results.push_back(static_verifier_resultt());
       auto &result = results.back();
 
-      if(e.is_true())
+      if(domain.is_bottom())
```

Comment:
I agree this seems to be an improvement, but it appears unrelated to the rest of the PR?

Reply:
Sort of.  The next commit refactors the same bit of code so I wanted to separate the 'improvement that changes the regression tests a bit' from the 'refactoring that adds new capabilities'.